analysis/webservice/webapp.py

```diff
@@ -156,6 +155,23 @@ def async_callback(self, result):
         if hasattr(result, 'cleanup'):
             result.cleanup()
 
+def inject_args_in_config(args, config):
+    """
+        Takes command argparse arguments and push them in the config
+         with syntax args.<section>_<option>
+    """
+    for _, t_opt in args._options.items():
```

Comment:
you can change this line to `for t_opt in args._options.values():`

Reply:
Ok, done